modin/pandas/test/test_io.py

```diff
@@ -352,135 +342,94 @@ def TestReadCSVFixture():
 
     yield
     # Delete csv files that were created
-    for filename in filenames:
-        if os.path.exists(filename):
-            try:
-                os.remove(filename)
-            except PermissionError:
-                pass
+    teardown_test_files(filenames)
 
 
-def setup_json_file(row_size, force=False):
-    if os.path.exists(TEST_JSON_FILENAME) and not force:
+def setup_json_file(filename=TEST_JSON_FILENAME, row_size=NROWS, force=True):
```

Comment:
Why `force=True`?

Reply:
Potentially, if default `filename` is used, setup_... functions are changed and old data file is still exists, these data files won't be replaced with new ones. With `force=True` we eliminate this possibility.

Reply:
May simply not provide a default value for `filename`?

Reply:
Ok, default value was removed.